src/core/core.h

```diff
@@ -108,6 +108,7 @@ typedef struct
     struct 
     {
         tic80_keyboard previous;
+        tic80_keyboard now;
```

Comment:
I just realized we should do the same for the `tic80_gamepads previous;` data

Reply:
I didn't think the gamepad could be used for studio control/input, so it's not a security risk, but it's easy enough to add to this PR if you like.  I'll do it.

Reply:
Gamepad used in `Surf` and `Game Menu`, so you also can hack them :)
Pls add the same for the gamepad too.
Thanks.